packages/esbuild/index.js

```diff
@@ -0,0 +1,117 @@
+let SizeLimitError = require('size-limit/size-limit-error')
+let { readdir } = require('fs').promises
+let { nanoid } = require('nanoid/non-secure')
+let { tmpdir } = require('os')
+let { join, resolve, parse } = require('path')
+let rm = require('size-limit/rm')
+
+let convertConfig = require('./convert-config')
+let runEsbuild = require('./run-esbuild')
+let getConfig = require('./get-config')
+
+const ESBUILD_EMPTY_PROJECT = 0
```

Comment:
We will need tests with empty project to validate these constants.

The project with empty file should have `0` size.

The project with one digit export (and `import` config) should have `1` size.

Reply:
I'm struggling with this a bit. Could you help me with that? Maybe we have an example of how to validate such kinds of constants

Reply:
Here is a test example for `*_EMPTY_PROJECT_IMPORT` and `*_EMPTY_PROJECT_IMPORT_GZIP`
https://github.com/ai/size-limit/blob/main/packages/webpack/test/index.test.js#L241-L268

Here is a test example for `*_EMPTY_PROJECT` and `*_EMPTY_PROJECT_GZIP` (yeap, it is in `size-limit` and not in `webpack` plugin for historical reason, feel free to move).
https://github.com/ai/size-limit/blob/main/packages/size-limit/test/run.test.js#L322-L328